docs/table/access_table.rst

```diff
@@ -178,7 +176,7 @@ Accessing Properties
 
 The code below shows accessing the table columns as a |TableColumns| object,
 getting the column names, table metadata, and number of table rows. The table
-metadata is an ordered dictionary (OrderedDict_) by default.
+metadata is an ordered dictionary (|OrderedDict|) by default.
```

Comment:
This one I'm not sure off; why not just `:class:`~collections.OrderedDict`? It does not seem to be used anywhere else, and we're attempting to remove its use...

Reply:
✅